meson.build

```diff
@@ -0,0 +1,38 @@
+project('komorebi', 'vala', 'c', version: '2.2.0', meson_version: '>=0.49', default_options: ['prefix=/usr'])
```

Comment:
Please put the kw arguments in new lines.

The default prefix should absolutely be `/usr/local`, which is what meson defaults to. You should never manually install anything to the `/usr` prefix, that is where your distro puts files :smile: